pkg/products/threescale/prometheusRules.go

```diff
@@ -286,7 +286,7 @@ func (r *Reconciler) newAlertReconciler(logger l.Logger, installType string, ctx
 							"sop_url": resources.SopUrlAlertsAndTroubleshooting,
 							"message": fmt.Sprintf("3Scale Zync has no pods in a ready state. Expected %d of pods.", r.Config.GetReplicasConfig(r.installation)["zyncApp"]),
 						},
-						Expr:   intstr.FromString(fmt.Sprintf("sum(kube_pod_status_ready{condition='true', namespace='%[1]v', pod=~'zync.*'} * on(pod, namespace) group_left() kube_pod_status_phase{phase='Running'}) < 1", r.Config.GetNamespace(), r.Config.GetReplicasConfig(r.installation)["zyncApp"])),
+						Expr:   intstr.FromString(fmt.Sprintf("sum(kube_pod_status_ready{condition='true', namespace='%[1]v', pod=~'zync.*', pod!~\"zync-database-.*\"} * on(pod, namespace) group_left() kube_pod_status_phase{phase='Running'}) < 1", r.Config.GetNamespace(), r.Config.GetReplicasConfig(r.installation)["zyncApp"])),
```

Comment:
Think we could follow the same syntax:
pod!~'zync-database-.*' instead of pod!~\"zync-database-.*\" ?
Also, might be good to extend the message to include that here we are talking about zync-queue and zync.

Reply:
Updated. Thank you